### DISCUSSION TITLE
Path nesting problem

### DISCUSSION THREAD

**Discussion by Kimishu** [juhaku/utoipa#1246] [Q&A] [answered]
I noticed one thing according to this [issue](https://github.com/tokio-rs/axum/issues/2659) (probably this will be fixed in 0.9 version)
When u create a route with name, for example, `/test` and then add an empty (`/`) get/post methods to it, utoipa makes it like `/test/` in swagger (and it's what I expected).
When I tried to execute it, I got 404 error, and only one route exists - `/test`. Cases "/" and "" (with empty string) will give the same result but with one exception - the first one will give 404 error in swagger.
Am I right in understanding that this is a [solution](https://github.com/juhaku/utoipa/pull/1231)? And it haven't been released yet?

**Comment by juhaku** [comment, marked as answer]
Yes, you are right, that PR makes utoipa axum nesting in par with the actual axum implementation so it should work the same as axum in all edge cases. It was not released until today.

**Comment by dr3s** [comment]
Using the latest releases but still seeing problems with nesting.  I can open a new issue but I'm not sure how to get nesting to work with collection resources.  I have an api like this:
"/api"
  - nest "/things" router
    -  "" get the list of things
    - "/{id}" get a thing
  
The router works if I drop the get a list of things.  Routes work when their not nested but still the same paths.  When using nesting I get:

`Overlapping method route. Handler for `GET /` already exists`

It seems like the "" get list of things is being interpreted as "/" and conflicting with the nested router itself.  What gives?
